src/com/google/common/css/compiler/gssfunctions/ColorUtil.java

```diff
@@ -28,6 +29,7 @@
   public static final int H = 0;
   public static final int S = 1;
   public static final int B = 2;
+  public static final int L = 2;
```

Comment:
Yes, that's it.
I'll add a comment before each (and to explain why B and L are both '2')